### PR TITLE
renv documentation added

### LINKED ISSUE #239: document how to work with renv
after merging #222, there is a new process to follow for how to work with renv. this needs to be documented somewhere.

The steps are:
* create a folder for the post/presentation you want to create
* the quarto doc will then be called index.qmd within that folder
* once you have created your doc (or, anytime you change the packages required), run `renv::snapshot("PATH_TO_THE_FOLDER")` (note, not the `index.qmd` but the folder)
* add near the top of the quarto doc (after the yaml) the following code chunk

```
` ` `{r lockfile}
#| include: FALSE
renv::use(lockfile = "renv.lock")
` ` `
```

(note there should not be spaces between the ` when you paste into quarto, but you can't display these in github)

### PR DESCRIPTION
**Pull request by yiwen-h**
Closes #239 
Does it make sense??

### CODE REVIEW

**Review by tomjemmett — CHANGES_REQUESTED** [review]
I think there are some more specific points to make, but on phone currently. Will update properly later!

File: README.md [hunk 1]
```diff
@@ -20,18 +20,52 @@ This repo features:
...
+To work on an already published blogpost/presentation:
+
+1. Run `renv::use(lockfile="/path/to/page/renv.lock")` in Console
+2. Edit the .qmd file that you are working on. To preview your changes, run `quarto preview path/to/page.qmd` in terminal.
 
 ### How to create a new presentation
 
-Make the presentation with quarto, and put it in `presentations/` in a `YYYY-MM-DD_Talk-title` folder. Your presentation should conform to the SU branding.
+1. Make the presentation with quarto, and put it in `presentations/` in a `YYYY-MM-DD_Talk-title` folder. Your presentation should conform to the SU branding. It should have the filename `index.qmd`
+2. If your presentation requires any specific packages, capture them with `renv::snapshot("PATH_TO_THE_FOLDER")` and ensure that you include in the top of your .qmd file (after the yaml) the following code chunk, without the hashes (#)
```
Comment (tomjemmett) [inline README.md:44]: for now, I think we need to always snapshot - need to investigate this more thoroughly (as R shouldn't be a dependency for this)

File: README.md [hunk 2]
```diff
@@ -20,18 +20,52 @@ This repo features:
...
 
 ### How to create a new blogpost 
 
 1. Navigate to the `blogs/posts` folder
-2. Copy a previous blogpost file and use that as your template
-3. If you want to see how it looks before pushing to GitHub, click Render in RStudio. The HTML version of your new post should open in your browser.
+2. Create a folder for your blogpost, following the naming convention `YYYY-MM-DD_title-of-post`
+3. Copy a previous blogpost index.qmd file into your folder and use that as your template
+4. Write your blogpost. To preview changes, run `quarto preview path/to/index.qmd` in terminal.
+5. If your blogpost requires any specific packages, capture them with `renv::snapshot("PATH_TO_THE_FOLDER")` and ensure that you include in the top of your .qmd file (after the yaml) the following code chunk, without the hashes (#)
```
Comment (tomjemmett) [inline README.md:60]: as above

**Review by tomjemmett — CHANGES_REQUESTED** [review]
see comments

**Review by tomjemmett — APPROVED** [review]

**Review by ChrisBeeley — APPROVED** [review]
I think the PR is great. I think I have some longer questions about whether relative novices are capable of contributing to this repo, but this is a great start to having better documentation

**Pull request merged**